### PR TITLE
Add rules of arbitration for representation limits

### LINKED ISSUE #83: Update charter with org-merge voting rules
From the discussion, the text we like most is:

> As an aside, note that this is more about the reputation of this group being co-opted in the future than it is about it being seen as a mouthpiece for a vested interest in the present. If it's obviously a mouthpiece, that's an issue, but it will impact our reputation and therefore adoption of our recommendations. If our group is co-opted in the future, however, then a group will be able to take advantage of our established reputation.
> 
> 1. Define representation groups that we wish to control representation on. For instance: one chair per independent organisation, 30% of community votes, 30% of tech leads (to avoid majorities of conspirators).
> 2. A organisation is defined as a single company.
> 3. If a member (define 'member') raises a concern of independence: first they must demonstrate a relationship between organisations that lead to the conclusion they should be treated as a single organisation (e.g ownership, contractor-contractee); second, the relationship will be debated, since this does not necessarily demonstrate that the organisations are acting together; and third, action will be taken only if a majority of uninvolved members (choose your membership set and quorum here) votes for it. The action would be that the multiple entities would be considered one organisation under the charter.
> 4. If, through this process, an organisation becomes overrepresented, the seats for that organisation will all be put up for a special election. The seats are open to all members, both previous holders and any other nominees.
> 5. In any elections (presuming we use STV or something similar), the members with least votes in an overrepresented group will be excluded from election (this should already be in some other part of the charter). Where there are multiple candidates with the same number of least votes, it's a coin toss (ditto).
> 6. The decision on independence or dependence may be revisited after a period of not less than three months. A member may raise it for discussion and the process in point 3 shall be repeated. No immediate re-election of members shall occur if the organisations now are deemed independent; this will be taken into consideration in the next natural election cycle.
> To game the above rules, you'd require a majority of the membership conspiring against some part of the community that had overrepresentation if and only if it were considered a bloc, and that bloc would need some corporate relationship, so it would be an unproductive thing to try and do unless a real concern existed. (I can't claim 'everyone not in my company is conspiring against me', for instance, because I haven't got valid grounds to raise the concern.)
> 
> It does not present multiple independent blocs agreeing to vote in concert. What prevents that is a healthy and diverse membership, and there's nothing the rules can do about that.
>

### PR DESCRIPTION
**Pull request by iawells**
The first change in this pull request also includes some cosmetics on the wording of the charter.

NB: I have not yet changed the 'one company, one vote' rule, since we're about to start an election cycle.

### CODE REVIEW

**Review by rabi-abdel — APPROVED** [review]

**Review by xmulligan — APPROVED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Comment by iawells** [comment]
Merged with the changes in the original since the original pull request.

File: charter.md [hunk 1]
```diff
@@ -132,4 +132,44 @@ For the purpose of this document, “Company” shall mean any entity (Company-A
...
+
+The CNF WG strives to maintain its editorial independence from the companies and organisations that provide its members.  To this end, this charter sets limits on the involvement any given entity might have in its operation, in order to maintain the integrity of its output as a consensus view of industry technical experts.
+
+To this end, representation from a single entity for a given role is limited, to avoid undue influence.  This section determines how companies, organisations and individual interested parties are determined to be independent of each other, and the way in which this affects elections.
+
+#### Determination of independence
+
+1. An entity consists of a single company or organisation, a related group of companies or organisations in common ownership, or an individual acting independently of their employment.
+
+2.  Entities may potentially not be independent if:
```
Comment (electrocucaracha) [inline charter.md]: nit: extra space
```suggestion
2. Entities may potentially not be independent if:
```

**Review by electrocucaracha — APPROVED** [review]

File: charter.md [hunk 2]
```diff
@@ -78,31 +78,31 @@ Deliverables
...
 #### Chairs
 
-The CNF WG will be co-chaired by 3 people: at least one co-chair from the Kubernetes (K8s) Community, one from the Service Provider (SP) Community and one from the CNF Developer (Dev) Community to have equal representation.
-- *Primary role of Chairs is governance of the group.*
-- The Chairs are responsible for ensuring that group meetings are facilitated effectively, while also engaging group members in leadership roles. Effective facilitation includes the following activities:
+The CNF WG will be co-chaired by 3 people - one co-chair from the Kubernetes (K8s) Community, one from the Service Provider (SP) Community and one from the CNF Developer (Dev) Community - in order to maintain equal representation.
+
+- *The primary role of Chairs is to run operations and the governance of the group.*
+- The Chairs are responsible for ensuring that group meetings are planned and facilitated effectively, while also engaging group members in leadership roles. Effective facilitation includes (but is not limited to) the following activities:
+    - setting the agenda for meetings
```
Comment (taylor) [inline charter.md]:
```suggestion
- The Chairs are responsible for ensuring that group meetings are planned and facilitated effectively, while also engaging group members in leadership roles. Effective facilitation includes (but is not limited to) the following activities:
```

Remove "setting the agenda for meetings"

File: charter.md [hunk 3]
```diff
@@ -132,4 +132,44 @@ For the purpose of this document, “Company” shall mean any entity (Company-A
...
+
+3. If a member raises a concern of that two entities are not independent, or, conversely, that a single entity should be considered as two separate entities, the member shall make their case and the co-chairs will consider it.
+
+4. If they deem the case valid, the relationship and its effects will be discussed by members.  The existence of a relationship does not necessarily demonstrate that the organisations are acting in concert.
+
+5. A vote of all uninvolved members will be held to determine whether or not the multiple parties should be considered one entity under the rules of this charter.
+
+#### Reconsidering the decision
+
+The decision on independence or dependence of entities may be revisited after a period of not less than three months.  After this time, any member may raise it for discussion via the above process.
```
Comment (taylor) [inline charter.md:162]: Where did the three month time limit come from?

**Review by taylor — CHANGES_REQUESTED** [review]

File: charter.md (hunk 2 above)
Comment (taylor) [inline charter.md]:
```suggestion
- *Primary role of Chairs is governance of the group.*
- The Chairs are responsible for ensuring that group meetings are facilitated effectively, while also engaging group members in leadership roles. Effective facilitation includes the following activities:
```

**Review by taylor — CHANGES_REQUESTED** [review]

File: charter.md (hunk 2 above)
Comment (taylor) [reply charter.md]: Update from https://github.com/cncf/cnf-wg/pull/98

Comment (iawells) [reply charter.md]: why?

File: charter.md (hunk 3 above)
Comment (iawells) [reply charter.md:162]: Arbitrary choice.

File: charter.md (hunk 2 above)
Comment (iawells) [reply charter.md]: ok, done in the other proposed change

**Review by taylor — APPROVED** [review]

**Pull request merged**